---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "EntandoApp"
    EntandoResourceNamespace: "my-namespace"
    EntandoApp: "my-app"
  name: "my-app-06936515"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "9d420fa5-7b71-4a0a-beb1-af4a7b8b0076"
action: "FAILED"
count: 1
firstTimestamp: "2021-06-07T15:40:05Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "1276729"
  uid: "9d420fa5-7b71-4a0a-beb1-af4a7b8b0076"
lastTimestamp: "2021-06-07T15:40:05Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Test Exception.\
  \ Fix the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
source:
  component: "entando-k8s-app-controller"
type: "Error"
